src/sentry/static/sentry/app/views/organizationIntegrations/index.tsx

```diff
@@ -1,387 +1,3 @@
-import compact from 'lodash/compact';
```

Comment:
@NisanthanNanthakumar Let's leave this component alone and have the router use a different component than this so we don't have a bunch of diffs.